rubble-nrf5x/src/radio.rs

```diff
@@ -84,6 +88,21 @@ pub struct BleRadio {
     rx_buf: Option<&'static mut PacketBuffer>,
 }
 
+/// Acknowledge the DSIABLED event
+macro_rules! acknowledge_disabled_event {
```

Comment:
Can't this be a method?

Reply:
The early-return of `None` makes that a bit unergonomic... Or do you have an idea how to handle that?

Reply:
You can use `?` with a method returning `Option<()>`

Reply:
Whoa, I totally missed that this is possible with options as well, and not just results :exploding_head: